components/org.wso2.carbon.identity.api.server.workflow.management/org.wso2.carbon.identity.api.server.workflow.management.common/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```

Comment:
update with the correct year

Reply:
fixed by :https://github.com/wso2/identity-api-server/pull/446/commits/0f139184981177695cfca48d0258684c173a847d